website_sale_product_description/models/product_template.py

```diff
@@ -7,7 +7,6 @@
 _logger = logging.getLogger(__name__)
 
 
-# configAuthorization.set('PRISMA f3d8b72c94ab4a06be2ef7c95490f7d3')
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
```

Comment:
We have already a Sale Description field in the sale order. please update `description_website` field string to "Description for Website"